ctapipe/instrument/camera.py

```diff
@@ -341,6 +344,7 @@ def from_table(cls, url_or_table, **kwargs):
             pix_y=tab['pix_y'].quantity,
             pix_area=tab['pix_area'].quantity,
             pix_type=tab.meta['PIX_TYPE'],
+            sampling_rate=u.Quantity(tab.meta["SAMP_RATE"], u.GHz),
```

Comment:
these are stored in FITS, and that keyword is longer than 8 characters so should be "SAMPRATE" or "SAMPFREQ" perhaps

Reply:
in the case that this info is missing, perhaps fall back to a default (until we update the input files), so things work as before: i.e.:
```py3
tab.meta.get("SAMP_RATE", u.Quantity(1,u.GHz))
```

Reply:
or better yet, issue a warning that the sampling rate is unknown and defaulting to 1.0 GHz

Reply:
once this works properly, we can just run `ctapipe-dump-instrument` on a file with all CTA cameras and use the outputs to update ctapipe-extra